stdlib/auth/src/main/ballerina/src/auth/outbound_auth_provider.bal

```diff
@@ -14,18 +14,21 @@
 // specific language governing permissions and limitations
 // under the License.
 
-# Represents the outbound Auth provider. Any type of implementation such as JWT and OAuth2 should be object-wise similar
-# to the `OutboundAuthProvider` object.
+# Represents the outbound Auth provider that could be used to authenticate external endpoints. Any type of
+# implementation such as JWT and OAuth2 should be object-wise similar to the `OutboundAuthProvider` object.
 public type OutboundAuthProvider abstract object {
 
-    # Generates a token for the outbound request.
+    # Generates a authentication token for the outbound request with outbound Auth providers such as JWT and OAuth2.
```

Comment:
Let's remove the example part, since it means like we have only JWT and OAuth2 outbound providers. WDYT?
```suggestion
    # Generates a authentication token for the outbound request with outbound Auth providers.
```